python/ray/setup-dev.py

```diff
@@ -71,6 +71,8 @@ def do_link(package, force=False, local_path=None):
         "--yes", "-y", action="store_true", help="Don't ask for confirmation."
     )
     args = parser.parse_args()
+    if not args.yes:
+        print("NOTE: Use '-y' to override all python files without confirmation.")
```

Comment:
Why do we have this?

Reply:
Because I have been using it for a while without knowing you can do -y. Many other frameworks give you this hint. Or do you think "y/n/all" could be better?

Reply:
I'm ok with this.